Lib9c/Action/HackAndSlash.cs

```diff
@@ -156,15 +159,22 @@ public override IAccountStateDelta Execute(IActionContext context)
             Log.Verbose("{AddressesHex}HAS get CostumeStatSheet: {Elapsed}", addressesHex, sw.Elapsed);
             sw.Restart();
 
-            if (avatarState.actionPoint < stageRow.CostAP)
+            if (playCount <= 0)
```

Comment:
we need check stage already cleared when playCount > 1

Reply:
Added. Please review again 🙏🏻 @ipdae